ArmaForces.Boderator/.gitignore

```
@@ -13,4 +13,5 @@
 Boderator/*
 BoderatorTest/*
 BoderatorWeb/*
-BoderatorWebTest/*
\ No newline at end of file
+BoderatorWebTest/*
+log*.txt
```

I think there should be a new line at the end of file.
```suggestion
log*.txt

```